package/CHANGELOG

```diff
@@ -53,6 +56,7 @@ Changes
     Bauhaus API. (Issue #857)
   * Qcprot now takes N x 3 arrays instead of 3 x N arrays. This gives about a
     40% speed up. (PR #930)
+  * The ProgressMeter class now has a `dynamic` keyword argument. (PR #944)
```

Comment:
Please also note that is has changed the formatting and now supports the `format` syntax as well and that is is preferred.